### PR TITLE
Add NHcommunity Moderators

### LINKED ISSUE #3: [FEAT] - Recognise Our Moderators
# Feature Request

## Is your feature request related to a problem? Please describe:

As a user, I should see a page which recognises the moderators of our community. This page should include information about each moderator, a link to their website/profile/whichever they'd like.

## Describe the solution you'd like

We should create a new file (`moderators.html`, maybe) within the `views` directory for this page.

### PR DESCRIPTION
**Pull request by Evavic44**
## Description
- Recognize moderators
- Add Moderators Section
- 

Add the moderators of NHcommunity

## Related Issue
#3 #5 

Closes #3

### CODE REVIEW

File: public/index.css [hunk 1]
```diff
@@ -142,3 +171,13 @@ li {
 		margin: auto;
 	}
 }
+
+@media(max-width: 870px) {
+   /* #moderators-section {
+      margin: 0 1rem;
+   } */
```
Comment (Akshu-on-github) [inline public/index.css:178]: Should we remove this if it has been commented out?

**Review by Akshu-on-github — CHANGES_REQUESTED** [review]
Here are a few things that I noticed:
- The moderators' social media links aren't filled in - might want to add those in before merging
- The moderators' pictures are uploaded to the branch - I'd suggest using a link to the GitHub Avatar so that it reflects any changes to the profile picture that may occur

File: public/index.css (hunk 1 above)
Comment (Evavic44) [reply public/index.css]: Removed. https://github.com/nhcommunity/homepage/pull/28/commits/2921eb1f09ed50a7c019b2fd365ef58055d9cee7

**Comment by Evavic44** [comment]
- Adding the links can be a `#goodfirstissue` for the community.
- That's a good idea. Can still be a good issue for the community. What do you think?

File: index.html [hunk 2]
```diff
@@ -68,39 +69,203 @@ <h2>Our Discussions</h2>
...
+                     </div>
+                  </div>
+            </div>
+
+            <div class="card">
+               <img src="./assets/moderators/David-Leal.png" alt="David Leal">
+               <div>
+                  <h3>David Leal</h3>
+                  <h4>Open source contributor, C/C++ & Lua coder</h4>
+                  <p>Maintainer @TheAlgorithms, @devicons</p>
```
Comment (Panquesito7) [inline index.html:187]:
```suggestion
                  <p>Maintainer @TheAlgorithms, @devicons, and 10+ open-source communities</p>
```

File: public/index.css [hunk 3]
```diff
@@ -142,3 +171,9 @@ li {
 		margin: auto;
 	}
 }
+
+@media(max-width: 870px) {
+   .moderators {
+      grid-template-columns: 1fr;
+   }
+}
```
Comment (Panquesito7) [inline public/index.css:179]: Missing endline.
```suggestion
}

```

**Comment by Akshu-on-github** [comment]
> * Adding the links can be a `#goodfirstissue` for the community.
> * That's a good idea. Can still be a good issue for the community. What do you think?

Sounds good to me 👍

**Comment by nhcarrigan** [comment]
For the record, not all of our Discord moderators are on GitHub.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
Here's my thoughts:

Rather than listing all of the information on the home page, I think we should have a cleaner display there with each moderator's:

- username
- avatar
- position in our org (moderator, support, developer)

And the entire card can be wrapped in a link to a page that contains their full information, using the moderator card provided in `templates`.

**Comment by Evavic44** [comment]
> 
> 
> For the record, not all of our Discord moderators are on GitHub.

Yeah, you mentioned that. They can be added subsequently on other PR's if possible.

**Comment by Evavic44** [comment]
I didn't notice there was a moderators file and I'm also kinda confused. Do you mean the individual cards of the moderators should be joined together and made a single link that redirects to the `moderators-card` page? @nhcarrigan 💡

**Comment by nhcarrigan** [comment]
image

I am thinking each mod gets a box like this on the home page, and clicking this box leads to the moderator's specific page (we can put these under `/mods/<username.html>`). Those moderator-specific pages would each use the template in `/templates` to display the moderator's information.

**Comment by Evavic44** [comment]
> 
> 
> image
> 
> I am thinking each mod gets a box like this on the home page, and clicking this box leads to the moderator's specific page (we can put these under `/mods/<username.html>`). Those moderator-specific pages would each use the template in `/templates` to display the moderator's information.
> 
> The subtext would be the member's role in our community - `support`, `moderator`, `developer`

Oh... that is clearer now. Will make the necessary changes. Thanks. 👍🏾

**Comment by Evavic44** [comment]
Changes implemented.
 
## Things to note
- Community roles may require updating since it was not specified what role each moderator has. 
- Discord moderators will need to be identified to be added.

File: index.html [hunk 4]
```diff
@@ -68,39 +69,137 @@ <h2>Our Discussions</h2>
...
+      <section id="moderators-section">
+         <div class="moderators-header">
+            <h2>Moderators</h2>
+            <p>Come meet our moderators and check out all the cool projects they’re working on and how they are helping the community.</p>
+         </div>
+         <div class="moderators">
+            <a href="templates/moderator-card.html" class="card">
+               <img src="https://avatars.githubusercontent.com/u/624760?v=4" alt="Eddiejaoude">
+               <div>
+                  <h3>Eddiejaoude</h3>
```
Comment (Panquesito7) [inline index.html:109]:
```suggestion
                  <h3>Eddie Jaoude</h3>
```

File: index.html [hunk 5]
```diff
@@ -68,39 +69,137 @@ <h2>Our Discussions</h2>
...
+                  <h4>Community Support</h4>
+               </div>
+            </a>
+
+            <a href="templates/moderator-card.html" class="card">
+               <img src="https://avatars.githubusercontent.com/u/63889819?v=4" alt="Nhcarrigan">
+               <div>
+                  <h3>Nhcarrigan</h3>
```
Comment (Panquesito7) [inline index.html:117]:
```suggestion
                  <h3>Nicholas Carrigan</h3>
```

File: index.html [hunk 6]
```diff
@@ -68,39 +69,137 @@ <h2>Our Discussions</h2>
...
+                  <h4>Community Manager</h4>
+               </div>
+            </a>
+
+            <a href="templates/moderator-card.html" class="card">
+               <img src="https://avatars.githubusercontent.com/u/62864373?v=4" alt="Vyvy-vi">
+               <div>
+                  <h3>Vyvy-vi</h3>
```
Comment (Panquesito7) [inline index.html:125]:
```suggestion
                  <h3>Vyom Jain</h3>
```

File: index.html [hunk 7]
```diff
@@ -68,39 +69,137 @@ <h2>Our Discussions</h2>
...
+                  <div>
+                     <h3>Shreyas44</h3>
+                     <h4>Community Developer</h4>
+                  </div>
+            </a>
+
+            <a href="templates/moderator-card.html" class="card">
+               <img src="https://avatars.githubusercontent.com/u/51391473?v=4" alt="Panquesito7">
+               <div>
+                  <h3>Panquesito7</h3>
```
Comment (Panquesito7) [inline index.html:149]:
```suggestion
                  <h3>David Leal</h3>
```

File: index.html [hunk 8]
```diff
@@ -68,39 +69,137 @@ <h2>Our Discussions</h2>
...
+                  <h4>Community Developer</h4>
+               </div>
+            </a>
+
+            <a href="templates/moderator-card.html" class="card">
+               <img src="https://avatars.githubusercontent.com/u/53310847?v=4" alt="Kaiwalyakoparkar">
+               <div>
+                  <h3>Kaiwalyakoparkar</h3>
```
Comment (Panquesito7) [inline index.html:157]:
```suggestion
                  <h3>Kaiwalya Koparkar</h3>
```

**Review by Panquesito7 — COMMENTED** [review]
These suggestions do not have to be accepted 🙂 if you like them, you can implement them.

**Comment by nhcarrigan** [comment]
I'm going to go ahead and rebase this and merge it in so I can play with the layout some more.

**Review by nhcarrigan — APPROVED** [review]
Looks good to me.

**Comment by nhcarrigan** [comment]
## Thank You

Thank you for your continued contribution to our project. We have reviewed your pull request and are happy to accept these changes.

Please continue to watch for issues labelled `help wanted`, as these will be additional opportunities to contribute.

**Comment by Evavic44** [comment]
> 
> ## Thank You
> 
> Thank you for your continued contribution to our project. We have reviewed your pull request and are happy to accept these changes.
> 
> Please continue to watch for issues labelled `help wanted`, as these will be additional opportunities to contribute.

Awesome. Thank you, Nick. 😎

**Pull request merged**